src/pages/Loans/LoansOverview/components/LoansTables/LoansTables.style.tsx

```diff
@@ -1,12 +1,37 @@
 import styled from 'styled-components';
 
-import { Flex } from '@/component-library';
+import { theme } from '@/component-library';
 
-const StyledTablesWrapper = styled(Flex)`
-  flex-direction: column;
+type StyledTablesWrapperProps = {
+  $hasPositions: boolean;
+};
+
+const StyledTablesWrapper = styled.div<StyledTablesWrapperProps>`
+  display: grid;
+  gap: ${theme.spacing.spacing6};
+  grid-template-areas: ${({ $hasPositions }) => {
+    if ($hasPositions) {
+      return `'lend-positions-table'
+              'lend-assets-table'
+              'borrow-positions-table' 
+              'borrow-assets-table'`;
+    }
+
+    return `'lend-assets-table'
+            'borrow-assets-table'`;
+  }};
 
   @media (min-width: 80em) {
-    flex-direction: row;
+    grid-template-columns: auto auto;
+    grid-template-rows: auto auto;
+    grid-template-areas: ${({ $hasPositions }) => {
+      if ($hasPositions) {
+        return `'lend-positions-table borrow-positions-table'
+                'lend-assets-table borrow-assets-table'`;
+      }
+
+      return "'lend-assets-table borrow-assets-table'";
```

Comment:
In one case you use backticks and in other double quotes, would be nice to unify - probably to backticks since these are used also in another file